bokeh/models/tiles.py

```diff
@@ -26,11 +26,11 @@ class TileSource(PlotObject):
     the maximum zoom level for the tile layer. This is the most "zoomed-in" level.
     """)
 
-    x_origin_offset = Float(default=20037508.34, help="""
+    x_origin_offset = Float(default=None, help="""
```

Comment:
I have mirrored the class hierarchy on the python side and set appropriate default there.  The only issue now is when running:
    git checkout sync-python-and-coffee
    git merge hot-fix/tile-renderer-initial-resolution
    cd bokehjs
    ./node_modules/gulp/bin/gulp.js scripts:generate

I get the following warnings about duplicate property definitions:

sys:1: RuntimeWarning: Properties {'y_origin_offset', 'initial_resolution', 'x_origin_offset'} were define
d in two classes, one of them <class 'bokeh.models.tiles.TileSource'>, as part of defining <class 'bokeh.m
odels.tiles.BBoxTileSource'>
sys:1: RuntimeWarning: Properties {'y_origin_offset', 'initial_resolution', 'x_origin_offset'} were define
d in two classes, one of them <class 'bokeh.models.tiles.TileSource'>, as part of defining <class 'bokeh.m
odels.tiles.QUADKEYTileSource'>
sys:1: RuntimeWarning: Properties {'y_origin_offset', 'initial_resolution', 'x_origin_offset'} were define
d in two classes, one of them <class 'bokeh.models.tiles.TileSource'>, as part of defining <class 'bokeh.m
odels.tiles.TMSTileSource'>
sys:1: RuntimeWarning: Properties {'y_origin_offset', 'initial_resolution', 'x_origin_offset'} were define
d in two classes, one of them <class 'bokeh.models.tiles.TileSource'>, as part of defining <class 'bokeh.m
odels.tiles.WMTSTileSource'>

Should the `defaults testing` check the object hierarchy?